ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -500,17 +500,20 @@ def install_client(master, client, extra_args=(),
     if password is None:
         password = client.config.admin_password
 
-    client.run_command(['ipa-client-install', '-U',
-                        '--domain', client.domain.name,
-                        '--realm', client.domain.realm,
-                        '-p', user,
-                        '-w', password,
-                        '--server', master.hostname]
-                       + list(extra_args))
+    result = client.run_command([
+        'ipa-client-install', '-U',
+        '--domain', client.domain.name,
+        '--realm', client.domain.realm,
+        '-p', user,
+        '-w', password,
+        '--server', master.Hostname] + list(extra_args)
```

Comment:
master.**H**ostname should be master.**hostname** instead.

Reply:
My mistake i will fix it, THANKS!